cpg_workflows/stages/exomiser.py

```diff
@@ -93,8 +94,10 @@ class CreateFamilyVCFs(DatasetStage):
 
     def expected_outputs(self, dataset: Dataset) -> dict[str, Path]:
         family_dict = find_families(dataset)
+        exomiser_version = config_retrieve(['workflow', 'exomiser_version'], 14)
```

Comment:
Would it be more legible to pull the magic number here and declare the version at the top of the file?

Reply:
I'm conscious not to introduce any more on-startup config reading, like the ones potentially solved [here](https://github.com/populationgenomics/production-pipelines/pull/691). Putting in a global method or a cached getter to retrieve this would reduce the number of times the value is grabbed, but it's just a dict value with no processing so I'm going with the inelegant but simple solution...

Unless that's me misunderstanding your comment